Description: Remove optional color-hex dependency
 The color-hex crate is not packaged in Debian, so this patch removes
 it as an optional dependency. This library is packaged as the binocle
 crate sub-dependency (ITP: #1111592), which does not require this
 feature at present. If color-hex becomes available in Debian in the
 future, this patch may be dropped.
Author: Nadzeya Hutsko <nadzya.info@gmail.com>
Forwarded: not-needed
Last-Update: 2025-09-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -68,9 +68,9 @@
 version = "0.3.1"
 optional = true
 
-[dependencies.color-hex]
-version = "0.2.0"
-optional = true
+# [dependencies.color-hex]
+# version = "0.2.0"
+# optional = true
 
 [dependencies.document-features]
 version = "0.2"
